doc/sphinxman/extract_changeset.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
`#!/usr/bin/env python3` for consistency with the rest, plus some systems do not have `python3` under `/usr/bin`.